src/gt4py/next/program_processors/runners/dace_iterator/__init__.py

```diff
@@ -254,21 +254,36 @@ def build_sdfg_from_itir(
     on_gpu: bool = False,
     column_axis: Optional[common.Dimension] = None,
     lift_mode: itir_transforms.LiftMode = itir_transforms.LiftMode.FORCE_INLINE,
+    skip_itir_lowering_to_sdfg: bool = False,
+    cache_id: Optional[str] = None,
+    disable_cache: bool = False,
 ) -> dace.SDFG:
     """Translate a Fencil into an SDFG.
 
     Args:
-        program:	        The Fencil that should be translated.
-        *args:		        Arguments for which the fencil should be called.
-        offset_provider:	The set of offset providers that should be used.
-        auto_optimize:	    Apply DaCe's `auto_optimize` heuristic.
-        on_gpu:		        Performs the translation for GPU, defaults to `False`.
-        column_axis:		The column axis to be used, defaults to `None`.
-        lift_mode:		    Which lift mode should be used, defaults `FORCE_INLINE`.
+        program:	                The Fencil that should be translated.
+        *args:		                Arguments for which the fencil should be called.
+        offset_provider:	        The set of offset providers that should be used.
+        auto_optimize:	            Apply DaCe's `auto_optimize` heuristic.
+        on_gpu:		                Performs the translation for GPU, defaults to `False`.
+        column_axis:		        The column axis to be used, defaults to `None`.
+        lift_mode:		            Which lift mode should be used, defaults `FORCE_INLINE`.
+        skip_itir_lowering_to_sdfg: Allows to use an SDFG from file, for debug only.
+        cache_id:                   The if of teh cache entry, used to disambiguate, caching on file.
+        disable_cache:              If set then the SDFG is not cached on disc, this disables the `skip_itir_lowering_to_sdfg` feature.
```

Comment:
I would not call it `disable_cache`, but rather `save_sdfg`.

Reply:
@edopao 
But `save_sdfg` does not go well with `skip...`. How if we would rename skip to `try_reload_sdfg` or something like that?

Reply:
Yes, maybe just `load_sdfg_from_file`